proto/p4/v1/p4runtime.proto

```diff
@@ -284,6 +284,23 @@ message Action {
 
 message ActionProfileActionSet {
   repeated ActionProfileAction action_profile_actions = 1;
+  // Support per-group hash modes and resource usage modes.
```

Comment:
This doesn't really make sense as a comment here. Can you reframe this to a comment each on ActionSelectionMode and SizeSemantics saying what they're both for?